src/components/Configure/layout/ConditionalProxyLayout.tsx

```diff
@@ -0,0 +1,25 @@
+import { LoadingIcon } from '../../../assets/LoadingIcon';
+import { useHydratedRevision } from '../state/HydratedRevisionContext';
+
+interface ConditionalProxyLayoutProps {
+  children: React.ReactNode;
+}
+
+/**
+ * if the hydratedRevision does not have read or write,
+ * then it will not render the ConfigureInstallation
+ * @returns
+ */
+export function ConditionalProxyLayout({ children }: ConditionalProxyLayoutProps) {
+  const { hydratedRevision, loading } = useHydratedRevision();
+  const isProxy = !hydratedRevision?.content.read && hydratedRevision?.content.write;
```

Comment:
to make this future-proof, we should not be explicitly checking for read and write keys, because we'll have other action types in the future (subscribe, etc.). You'll probably want something like this:

```
const keys = Object.keys(hydratedRevision?.content || {})
isProxy = keys.length === 1 && keys[0] === "proxy"
```

Reply:
will do this in a followup.

Reply:
Oh that's a great point, my idea won't work. It might make sense to keep a list in `constants.ts` for the non-proxy actions (for now, it'll be ["write", "read"]) and check the keys against that list.